specification/resources/resource-manager/Microsoft.Resources/stable/2019-10-01/resources.json

```diff
@@ -3314,7 +3314,8 @@
           "Tags"
         ],
         "operationId": "Tags_DeleteValue",
```

Comment:
Noun_Verb.

Reply:
This is the existing problematic operation ID which I cannot change due to SDK etc. Due to this, the other names are also affected.

Reply:
Ideally this should have been "PredefinedTagValues_Delete" etc.

Reply:
Generally it should match the last segment entity name, in this case would be TagValues_Delete

Reply:
Right, but changing it now is not possible due to its consumption by the SDKs.

Reply:
We uses operationid to determine method groups/methods in CLI and Powershell code generation. So it is important to get it to be correct and descriptive in order to avoid possible code generation issues.

Reply:
Per our conversation, update to use Tags_[Verb]AtScope, as used in operations like Deployments_DeleteAtScope etc.